ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AggregatingAttestationPool.java

```diff
@@ -68,7 +68,11 @@ public class AggregatingAttestationPool implements SlotEventsChannel {
    * <p>A limit of 40,000 attestations is enough for 3 slots worth at 400,000 validators which gives
    * a sane upper limit while still being above the typical 10-20k pool size seen on MainNet.
    */
-  public static final int DEFAULT_MAXIMUM_ATTESTATION_COUNT = 40_000;
+  /**
```

Comment:
Is this supposed to be part of the comment block above?

Also, we probably don't want to mention the 40k limit anymore.

Reply:
the logic was the basis for the following sttatement, i can try to clean it up more i was just being lazy 🗡️